config/nginx.conf

```diff
@@ -13,6 +13,7 @@ server {
 
     location ~ \.php$ {
         include snippets/fastcgi-php.conf;
+        fastcgi_read_timeout 45s;
```

Comment:
This does not seem to be related to this PR and being hardcoded here breaks the usage of the `/etc/nginx/conf.d/*.conf` where settings can easily be overriden

Reply:
While waiting for a better fix, here is a workaround for docker-compose
```yml
    ...
    volumes:
      # Contains own values for e.g. timeouts
      - ./rss-bridge/nginx.conf:/etc/nginx/conf.d/rss-bridge.conf:ro
    entrypoint:
      - /bin/sh
      - -c
      - |
        # Disable RSS-Bridge hard-coded timeout values, if any (45s is much too short for some bridges)
        sed -r -i '/^\s*.*timeout.*$$/s/^/#/' /etc/nginx/sites-available/default
        exec /app/docker-entrypoint.sh
```

Reply:
the default timeout in nginx is 60 and i think 45s is more sensible.

i didnt think of users being able to tweak this in the docker container by putting files in /etc/nginx/conf.d/*.conf.

well i can remove that line in the location context sure